src/pretalx/mail/signals.py

```diff
@@ -37,3 +37,25 @@
 contain the event. Additionally, the ``mail`` keyword argument contains
 the ``QueuedMail`` instance itself.
 """
+
+html_after_mail_badge = EventPluginSignal()
+"""
+This signal is sent out to display additional information related to emails in
+the internal organiser area.
+
+As with all plugin signals, the ``sender`` keyword argument will contain the
+event. Additionally, the signal will be called with the ``request`` it is
+processing, and the ``mail`` which is currently displayed.
+The receivers are expected to return HTML.
+"""
+
+html_below_mail_subject = EventPluginSignal()
+"""
+This signal is sent out to display additional information related to emails in
+the internal organiser area.
```

Comment:
This is not correct as implemented – the `mail_log.html` file is show to the speaker aswell.

Reply:
That is right! - And should not necessarily be that way.
There obviously is already a differentiation, because the 'accordion' cannot be opened on the public profile page.
Makes a lot of sense, not to add the information there - although it might also make at least some sense to let the plugin decide.

Reply:
Minor additional remart: the accordion doesn't open in the public view but the HTML is the same as in the organiser backend.

Reply:
I have changed this and moved the hooks to inject HTML and/or forms into the email editor.
Tje hooks are not on the new email composition page now. I'm not yet sure if I also want them there.